devise.gemspec

```
@@ -14,6 +14,13 @@ Gem::Specification.new do |s|
   s.homepage    = "https://github.com/heartcombo/devise"
   s.description = "Flexible authentication solution for Rails with Warden"
   s.authors     = ['José Valim', 'Carlos Antônio']
+  s.metadata    = {
+    "homepage_uri"     => "https://github.com/heartcombo/devise",
+    "changelog_uri"    => "https://github.com/heartcombo/devise/blob/master/CHANGELOG.md",
+    "source_code_uri"  => "https://github.com/heartcombo/devise/issues",
```

This should be either: 

```suggestion
    "source_code_uri"  => "https://github.com/heartcombo/devise",
```

Or:

```suggestion
    "bug_tracker_uri"  => "https://github.com/heartcombo/devise/issues",
```